packages/serializer/addon/-private/embedded-records-mixin.js

```diff
@@ -94,9 +94,9 @@ import { warn } from '@ember/debug';
   to modify it to fit your specific needs.**
 
   For example, review the docs for each method of this mixin:
-  * [normalize](/api/data/classes/DS.EmbeddedRecordsMixin.html#method_normalize)
-  * [serializeBelongsTo](/api/data/classes/DS.EmbeddedRecordsMixin.html#method_serializeBelongsTo)
-  * [serializeHasMany](/api/data/classes/DS.EmbeddedRecordsMixin.html#method_serializeHasMany)
+  * [normalize](/api/data/classes/EmbeddedRecordsMixin.html#method_normalize)
```

Comment:
these links are likely still broken (and were likely broken before) as the API docs url structure is not `/api/data/classes/` but `/ember-data/release/classes/`

Reply:
Correct, any sub header I navigate to uses `?anchor`, e.g. [`registerForType`](https://api.emberjs.com/ember/3.12/classes/Application/methods/registerOptionsForType?anchor=registerOptionsForType)